### DISCUSSION TITLE
Does LightBulb have 【command line】？

### DISCUSSION THREAD

**Discussion by k22725** [Tyrrrz/LightBulb#318] [Q&A]
Does LightBulb have command line functionality similar to twinkle-tray
( https://github.com/xanderfrangos/twinkle-tray?tab=readme-ov-file#command-line-arguments )?

**Comment by Tyrrrz** [comment]
No, it doesn't

**Comment by e-t-l** [comment]
This is not an efficient solution, but it is a workaround for as long as there is no CLI support: You could assign hotkeys in Lightbulb, then write a simple autohotkey script to simulate a keypress of the desired hotkey when called from a command line.
